frontend/controller/namespace.js

```diff
@@ -1,11 +1,12 @@
+// @no-flow
 'use strict'
 
 import sbp from '@sbp/sbp'
 import { handleFetchResult } from './utils/misc.js'
 
 // NOTE: prefix groups with `group/` and users with `user/` ?
 sbp('sbp/selectors/register', {
-  'namespace/register': (name: string, value: string) => {
+  'namespace/register': (name, value) => {
```

Comment:
I had to remove Flow annotations here and also in giLodash because Deno choked on them. Converting these files to .ts was another option, but then the frontend would be in a mixed state where most files are .js while a few are .ts.

I can add JSDoc comments for now, but I think we'll still want to re-add "real" annotations later.